doc/api/stream.md

```diff
@@ -428,15 +428,13 @@ changes:
   `Uint8Array`. For object mode streams, `chunk` may be any JavaScript value
   other than `null`.
 * `encoding` {string} The encoding if `chunk` is a string
-* `callback` {Function} Optional callback for when the stream finishes
-  or errors
+* `callback` {Function} Callback for when the stream is finished.
```

Comment:
Does this mean it's not optional anymore?

Reply:
Just bringing it inline with the docs for write. Maybe should leave optional here and add it to write as well?